requirements.txt

```diff
@@ -70,7 +70,7 @@ semantic-version~=2.8.5
 simple-chalk~=0.1.0
 six~=1.15.0
 sqlparse~=0.4.1
-stripe~=2.56.0
```

Comment:
3.x has breaking changes, 4.x too. 

We should pin one version and not leave it totally open. 

@phot0n review this?


https://github.com/stripe/stripe-python/blob/master/CHANGELOG.md